### PR TITLE
edit readme.md

### PR DESCRIPTION
**Pull request by SergSenras**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -209,3 +209,8 @@ My name is Vanya, and I am glad to be part of the team.
 
 ![cat-forking](https://i.imgur.com/6eoCVvc.jpg)
 
+
+-----------------------------------------------------------------
+
+## Hello everybody!
+![funny cat](gif/Funny-Cat-Gifs-11.gif)
```
Comment (OleksiyRudenko) [inline README.md]: It is important to have an empty line at the end of a file under CVS. Please, add one, commit, then push.

**Comment by SergSenras** [comment]
Thanks for your notes.
I added empty line but commit diff now shows "1 addition & 1 deletion". Not sure this is correct ...

**Comment by OleksiyRudenko** [comment]
@SergSenras Seems like your editor is set to remove newline ending.
Just curious, what if you add two empty lines?

**Comment by OleksiyRudenko** [comment]
@SergSenras Perfect, thanks!
Merger approved.

**Comment by SergSenras** [comment]
Oh, look like It helped
Thanks!


> @SergSenras Seems like your editor is set to remove newline ending.
> Just curious, what if you add two empty lines?

It helped, thanks!

**Pull request merged**